docs/DeveloperGuide.md

```diff
@@ -885,21 +882,53 @@ testers are expected to do more *exploratory* testing.
 
     1. Download the jar file and copy into an empty folder
 
-    1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be
-       optimum.
+    1. Launch the jar file using the java -jar command rather than double-clicking (reason: to ensure the jar file is 
```

Comment:
I think we can suggest to double click then if it doesn't work, user can alternatively open their terminal in the directory where the jar file is and enter the command java <jar file name>-jar

Reply:
I copied this from our module recommendation, not sure which should be the correct situation

Reply:
ohh okok then follow the recommendation